packages/frontend/src/components/dialogs/ConfirmDeleteMessage.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+
+import Dialog, {
+  DialogBody,
+  DialogContent,
+  DialogFooter,
+  FooterActions,
+} from '../Dialog'
+import FooterActionButton from '../Dialog/FooterActionButton'
+import useTranslationFunction from '../../hooks/useTranslationFunction'
+
+import type { DialogProps } from '../../contexts/DialogContext'
+import { BackendRemote, Type } from '../../backend-com'
+import { C } from '@deltachat/jsonrpc-client'
+
+export type Props = {
+  accountId: number
+  msg: Type.Message
+  canSend: boolean
+} & DialogProps
+
+export default function ConfirmDeleteMessageDialog(props: Props) {
+  const tx = useTranslationFunction()
+
+  const { accountId, msg, onClose } = props
+
+  const deleteForAllPossible =
```

Comment:
Also, are we supposed to be able to delete info messages for all?

Android doesn't allow this: https://github.com/deltachat/deltachat-android/pull/3633/files#diff-200a4a1e945e1f2cf60ed07d8b3cf56c974ced107f47cf4ff60e0ebe993abc4cR69

Reply:
I will filter by isInfo